reactive/flannel.py

```diff
@@ -144,6 +146,8 @@ def reconfigure_flannel_service():
 def etcd_changed(etcd):
     if data_changed('flannel_etcd_connections', etcd.get_connection_string()):
         remove_state('flannel.service.installed')
+    if data_changed('flannel_etcd_client_cert', etcd.get_client_credentials()):
+        remove_state('flannel.service.installed')
```

Comment:
This will lead to a restart of flannel, but it won't lead to the etcd certs being written to disk again (which happens [here](https://github.com/charmed-kubernetes/charm-flannel/blob/35ee04c85ddb0b4b31ac7ee55eebf6844da2e6f2/reactive/flannel.py#L79)).

Reply:
This should be addressed with the latest commit.